openmdao/visualization/n2_viewer/index.html

```diff
@@ -2,82 +2,201 @@
     <div id="ptN2ContentDivId">
         <!-- The Model -->
         {{help}}
-        <div id="toolbarLoc">
-            {{toolbar}}
-            <div style="margin-top:5px; display:inline-block;">
-                <input type="text" id="awesompleteId" size="80" />
-                <button class="myButton" id="searchButtonId" title="Search"><i class="icon-search"></i></button>
-                <label id="searchCountId" style="display:inline-block;width:120px"></label>
+
+        <!-- Toolbar -->
+        <div>
+            <div id="toolbarLoc">
+                <img src="https://i.imgur.com/DU1lNQO.png" id="company-logo" />
```

Comment:
does this external link also need to be removed?

Reply:
That can be left there it's just a placeholder for the openMDAO logo, I do not have the actual one yet so it doesn't matter if it's visible or not.